[2021-08-10 19:00:33,365] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: Covid.Covidtask5 2021-08-10T13:14:01.532369+00:00 [queued]>
[2021-08-10 19:00:33,395] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: Covid.Covidtask5 2021-08-10T13:14:01.532369+00:00 [queued]>
[2021-08-10 19:00:33,396] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-08-10 19:00:33,396] {taskinstance.py:1068} INFO - Starting attempt 1 of 8
[2021-08-10 19:00:33,396] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-08-10 19:00:33,491] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): Covidtask5> on 2021-08-10T13:14:01.532369+00:00
[2021-08-10 19:00:33,496] {standard_task_runner.py:52} INFO - Started process 36527 to run task
[2021-08-10 19:00:33,505] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'Covid', 'Covidtask5', '2021-08-10T13:14:01.532369+00:00', '--job-id', '361', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp429dy641', '--error-file', '/tmp/tmpwkb8e7xe']
[2021-08-10 19:00:33,507] {standard_task_runner.py:77} INFO - Job 361: Subtask Covidtask5
[2021-08-10 19:00:33,637] {logging_mixin.py:104} INFO - Running <TaskInstance: Covid.Covidtask5 2021-08-10T13:14:01.532369+00:00 [running]> on host deepika-Inspiron-5570
[2021-08-10 19:00:33,860] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=sthadpka93@gmail.com
AIRFLOW_CTX_DAG_OWNER=postgres
AIRFLOW_CTX_DAG_ID=Covid
AIRFLOW_CTX_TASK_ID=Covidtask5
AIRFLOW_CTX_EXECUTION_DATE=2021-08-10T13:14:01.532369+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-08-10T13:14:01.532369+00:00
[2021-08-10 19:00:34,957] {python.py:151} INFO - Done. Returned value was: None
[2021-08-10 19:00:34,961] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=Covid, task_id=Covidtask5, execution_date=20210810T131401, start_date=20210810T131533, end_date=20210810T131534
[2021-08-10 19:00:35,042] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-08-10 19:00:35,078] {local_task_job.py:151} INFO - Task exited with return code 0
